appveyor.yml

```diff
@@ -0,0 +1,40 @@
+environment:
+  matrix:
+
+    - PYTHON: "C:\\Python36"
+      PYTHON_VERSION: "3.6.3"
+      PYTHON_ARCH: "32"
+    - PYTHON: "C:\\Python36-x64"
+      PYTHON_VERSION: "3.6.3"
+      PYTHON_ARCH: "64"
+
+    - PYTHON: "C:\\Python37"
+      PYTHON_VERSION: "3.7.0"
+      PYTHON_ARCH: "32"
+    - PYTHON: "C:\\Python37-x64"
+      PYTHON_VERSION: "3.7.0"
+      PYTHON_ARCH: "64"
+
+install:
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "python --version"
+
+  # Upgrade to the latest version of pip to avoid it displaying warnings
+  # about it being out of date.
+  - "python -m pip install --disable-pip-version-check --user --upgrade pip"
+
+  # Setting up enviroment
+  - "pip install poetry"
+  - "poetry install"
+  - "poetry develop"
+
+build: off
+
+test_script:
+  - "poetry run pytest"
+  - "poetry run mypy wemake_python_styleguide"
+  - "poetry run doc8 -q docs"
+
+on_success:
```

Comment:
I'm not sure about this. Do we really need to check it in both CI- on Travis and AppVeyor?

Reply:
Now, let's leave it to travis.